bloomfilter-blocked.cabal

```diff
@@ -128,13 +138,9 @@ benchmark bench
     , criterion
     , random
 
--- It's not really a test suite, but if we make it an executable then its
--- dependencies will be included for dependency resolution when building the
--- main library. As a test-suite, it's more accurately represented as an
--- internal component.
-test-suite fpr-calc
+executable fpr-calc
   import:         language, warnings
-  type:           exitcode-stdio-1.0
+  scope:          private
   hs-source-dirs: tests
   main-is:        fpr-calc.hs
   build-depends:
```

Comment:
I had changed these to test suites because `executable`'s dependencies are used in dependency resolution *even if* `scope: private`. Isn't it nicer that way?